docs/faq/faq.md

```diff
@@ -40,3 +40,11 @@ There are some things you should consider before doing so:
   conflicts.
 
 TODO: Link to adoption use case once it gets on board.
+
+## Why Cluster API doesn't have the CRD yaml file?
+
+Kubernetes provides two methods to extend APIs: Custom Resource and Kubernetes API Aggregation Layer, for more detail, we can refer to [Extending the Kubernetes API](https://kubernetes.io/docs/concepts/extend-kubernetes/).
```

Comment:
```suggestion
Kubernetes provides two methods to extend APIs: Custom Resource and the Kubernetes API Aggregation Layer. For more details, you can refer to [Extending the Kubernetes API](https://kubernetes.io/docs/concepts/extend-kubernetes/).
```
In general, when referring to the reader or user, it is common to use "you."
On the other hand, "we" typically represents the developers or authors.
Therefore, it is preferred to use "you" when explaining concepts or instructions to the reader.

Reply:
Thanks a lot~